depdency-injection/1.0/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Jakarta Depdency Injection 1.0"
+date: 2019-07-27T15:57:57-0700
+summary: "Final for Jakarta EE 8"
+---
+
+Jakarta Dependency Injection specifies a means for obtaining objects in such a way as to maximize reusability, testability and maintainability compared to traditional approaches such as constructors, factories, and service locators (e.g., JNDI).
+
+
+# Ballots
+
+## Release Review
+
+|                       |  Yes    | No      | Abstain  |
+|-----------------------|---------|---------|----------|
+|Fujitsu                |         |         |          |
+|IBM                    |         |         |          |
+|Oracle                 |         |         |          |
+|Payara                 |         |         |          |
+|Red Hat                |         |         |          |
+|Tomitribe              |         |         |          |
+|EE4J PMC               |         |         |          |
+|Participant Members    |         |         |          |
+|Committer Members      |         |         |          |
+
+# Compatible Implementations
+
+* https://weld.cdi-spec.org/download/[Weld 2.3.8.Final]
```

Comment:
Any reason to list this old version here? 2.3 has been dead for three years now.
I'd go with the up to date one which also implements CDI 2.0, which is currently Weld 3.1+ (3.1.1.Final ATM, 3.1.2.Final within a week's time).